crates/serde_spanned/Cargo.toml

```diff
@@ -26,3 +26,9 @@ pre-release-replacements = [
 
 [dependencies]
 serde = { version = "1.0.145", optional = true }
+
+[dev-dependencies]
+serde = "1"
+serde_derive = "1"
+serde-untagged = "0.1"
+toml = { version = "0.8", path = "../toml" }
```

Comment:
This introduces a circular dependency which will prevent publishing.

If you drop `version`, everything should work

Reply:
Good call! Done.